### PR TITLE
Direct URL for triggering a pipeline with SKIP_DOCKER=false.

### PR DESCRIPTION
**Pull request by Zimmi48**
**Kind:** documentation

### CODE REVIEW

**Review by SkySkimmer — APPROVED** [review]

File: dev/ci/README-developers.md [hunk 1]
```diff
@@ -179,6 +179,8 @@ but if you wish to save more time you can skip the job by setting
 
 This means you will need to change its value when the Docker image
 needs to be updated. You can do so for a single pipeline by starting
-it through the web interface.
+it through the web interface. Here is a direct link that you can use
+to trigger such a build:
+`https://gitlab.com/coq/coq/pipelines/new?var[SKIP_DOCKER]=false&ref=pr-XXXXX`
```
Comment (SkySkimmer) [inline dev/ci/README-developers.md:184]: Maybe add a note that you'll get a 404 page if unauthenticated.

Comment (Zimmi48) [reply dev/ci/README-developers.md:184]: Good point, will do.

**Comment by Zimmi48** [comment]
Updated.

File: dev/ci/README-developers.md [hunk 2]
```diff
@@ -179,6 +179,11 @@ but if you wish to save more time you can skip the job by setting
 
 This means you will need to change its value when the Docker image
 needs to be updated. You can do so for a single pipeline by starting
-it through the web interface.
+it through the web interface. Here is a direct link that you can use
+to trigger such a build:
+`https://gitlab.com/coq/coq/pipelines/new?var[SKIP_DOCKER]=false&ref=pr-XXXXX`.
+Note that this link will give a 404 error if you are not logged in or
+a member of the Coq organization on GitLab.  To request to join the
+Coq organization, go to https://gitlab.com/coq to request access.
```
Comment (jfehrle) [inline dev/ci/README-developers.md:187]:
```suggestion
it through the web interface. You can do so for a single pipeline by manually
starting a job using the green "Run pipeline" button on the
[web interface](https://gitlab.com/coq/coq/pipelines) with the
`SKIP_DOCKER` environment variable set to `false`.

Or, alternatively, use this direct link to trigger such a build:
`https://gitlab.com/coq/coq/pipelines/new?var[SKIP_DOCKER]=false&ref=pr-XXXXX`.
Note that this link will give a 404 error if you are not logged in or
a member of the Coq organization on GitLab.  To request to join the
Coq organization, go to https://gitlab.com/coq to request access.
```

**Review by jfehrle — COMMENTED** [review]
I would mention both ways to trigger the job.  The direct URL API is rather low-level and you don't see any status before you start it (e.g. it's already running).

**Comment by SkySkimmer** [comment]
> The direct URL API is rather low-level and you don't see any status before you start it (e.g. it's already running).

I have no clue what you mean, for me it's just like clicking the button and getting the SKIP_DOCKER prefilled.

**Comment by Zimmi48** [comment]
It cannot hurt to have both ways documented. I've committed the suggestion.

**Comment by SkySkimmer** [comment]
AFAICT they're the same way

**Comment by Zimmi48** [comment]
From the point of view of the computer yes, but not from the point of view of the user. Do you feel that the previous version of the PR was better?

**Comment by SkySkimmer** [comment]
It's less verbose so yes

**Comment by Zimmi48** [comment]
The PR is back to the previous version.

**Comment by jfehrle** [comment]
> > The direct URL API is rather low-level and you don't see any status before you start it (e.g. it's already running).
> 
> I have no clue what you mean, for me it's just like clicking the button and getting the SKIP_DOCKER prefilled.

If you start the job through the `https://gitlab.com/coq/coq/pipelines` page, you can see whether it's already running.  If you're updating the Docker info, Coqbot will submit a useless job that will fail that can be cancelled.

And BTW, the doc doesn't mention substituting the PR number for XXXXX in the link.

**Pull request merged**